src/modules/sendAPI.js

```diff
@@ -0,0 +1,13 @@
+async function sendAPi() {
+  const send = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+  fetch(send, {
+    method: 'POST',
+    body: JSON.stringify({
+      // name: 'Beyo',
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  }).then((param) => param.json()).then((saved) => saved);
+}
```

Comment:
- [ ] kindly change this to arrow functions instead of the function keyword.

Reply:
well resolved! ✔️